web/cypress/integration/route/create-edit-route-with-redirect-plugin.spec.js

```diff
@@ -31,6 +31,7 @@ context('Create Edit and Delete Route with redirect plugin', () => {
     deleteAlert: '.ant-modal-body',
     notificationCloseIcon: '.ant-notification-close-icon',
     notification: '.ant-notification-notice-message',
+    webSocketSelector: '[title=WebSocket]',
```

Comment:
it would be better to assert both websocket label and switch are not existed

Reply:
Done ~ @liuxiran